app/Http/Controllers/Multiplayer/Rooms/Playlist/ScoresController.php

```diff
@@ -66,7 +66,10 @@ public function index($roomId, $playlistId)
         $scoresJson = json_collection(
             $highScores->pluck('scoreLink'),
             $transformer,
-            ScoreTransformer::MULTIPLAYER_BASE_INCLUDES
+            [
+                ...ScoreTransformer::MULTIPLAYER_BASE_INCLUDES,
+                'position',
```

Comment:
I can _maybe_ skip this one and hardcode a 1..50 position client-side if it helps.

Reply:
yeah this will cause as many additional queries as there are scores

Reply:
Have force-pushed this out.